Feature/150 pmsmart clear activities (#162)
itsallcode/white-rabbit: README.md

    1. Select menu Reports > Project report
    1. Click the "Export to pmsmart" button

#### Optional configuration settings

Optionally you can configure pmsmart plugin to skip transfer of a comment for each activity. 

For each activity in WhiteRabbit you can enter a comment. By default pmsmart plugin transfers these comment to pm-smart. As the web ui is quite slow, transfer of comments can take a while. If you want to speed-up pm-smart export by skipping transfer of comments you can add an optional property to WhiteRabbit configuration file.

    ```properties
    pmsmart.transfer.comments = false
    ```

Optionally you can configure pmsmart plugin to clear durations for all other projects, not mentioned as activity in WhiteRabbit.

> For each day pmsmart plugin by default transfers the durations of all activities entered into WhiteRabbit. If pm-smart contains durations for other projects then these are not overwritten. This will especially happen if user transfers his or her time-recording for a particular month multiple times and changes the selection of activities in between. In order to ensure consistent data in pm-smart you can add an optional property to WhiteRabbit configuration file.

Reviewer: ```suggestion
For each day pm-smart plugin by default transfers the durations of all activities entered into WhiteRabbit. If pm-smart contains durations for other projects then these are not overwritten. This will especially happen if users transfer their time-recording for a particular month multiple times and changes the selection of activities in between. In order to ensure consistent data in pm-smart you can add an optional property to WhiteRabbit configuration file:
```
